android/settings.gradle

```diff
@@ -1,5 +1,4 @@
 rootProject.name = 'sentry_react_native'
-include ':react-native-linear-gradient'
-project(':react-native-linear-gradient').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-linear-gradient/android')
```

Comment:
The latest version of `react-native-linear-gradient` doesn't mention that this is needed.